SS_versioninfo_330safe.tpl

```diff
@@ -5,9 +5,13 @@ DATA_SECTION
 !!//  SS_Label_Section_1.0 #DATA_SECTION
 
 !!//  SS_Label_Info_1.1.1  #Create string with version info
-!!version_info+="#V3.30.17.01Beta;_2021_08_24;_safe;_Stock_Synthesis_by_Richard_Methot_(NOAA)_using_ADMB_12.3";
```

Comment:
Maybe this is a thought for another time, but I wonder if it would be possible to just have SS_versioninfo_330.tpl and get rid of the 3 different versions? The headers are nearly identical except for 1 word and it would reduce the need to hand edit 3 different files. Could just open a new issue to address this.